src/main/kotlin/ch/srgssr/pillarbox/monitoring/event/model/DeviceNameProcessor.kt

```diff
@@ -0,0 +1,55 @@
+package ch.srgssr.pillarbox.monitoring.event.model
+
+import ch.srgssr.pillarbox.monitoring.io.readLines
+import org.springframework.core.io.DefaultResourceLoader
+
+/**
+ * Processor that maps device model identifiers to human-readable device names.
+ *
+ * This processor loads a mapping from a resource file `classpath:mappings/device-name.csv`
+ * and applies it to incoming event data.
+ */
+class DeviceNameProcessor : DataProcessor {
+  companion object {
+    /**
+     * Map of device models to human-readable device names.
+     * Source: https://gist.github.com/adamawolf/3048717
+     */
+    private val mappings: Map<String, String> =
+      DefaultResourceLoader().readLines("classpath:mappings/device-name.txt") { lines ->
+        lines
+          .map { it.trim() }
+          .filter { it.isNotBlank() }
+          .filter { ':' in it }
```

Comment:
It's enough to do the following, no? If the line is blank, it won't contain `:`.
```suggestion
          .filter { ':' in it }
```